finance/bitcoin/Portfile

```diff
@@ -12,7 +12,7 @@ legacysupport.use_mp_libcxx \
 
 name                    bitcoin
 categories              finance crypto
-version                 25.0
+version                 25.1
 revision                0
 platforms               darwin
```

Comment:
`platforms darwin` is implicit, we can drop this line

Reply:
Dropped this line.